### DISCUSSION TITLE
Education Content Creation Guideline Idea (blog and video series)

### DISCUSSION THREAD

**Discussion by Lukisong82** [chainbase-labs/manuscript-core#46] [Ideas]
# Manuscript: A Guide to Blockchain Data Exploration

## Introduction to Manuscript
This section introduces Manuscript as a tool for accessing and querying blockchain data, highlighting its ability to simplify the process.

## Initializing Manuscript

1. Using the Init Command: Describes the initialization process of Manuscript and the local environment using the init command, allowing users to select their desired blockchain data and storage options.
2. Selecting Blockchain Data: Guides users through selecting specific blockchain data tables from the available options.
3. Choosing Target Storage: Explains the process of selecting a storage option for Manuscript to process data, highlighting PostgreSQL as an example.
4. Confirmation and Initialization: Outlines the final steps of initialization, confirming selections and triggering the setup of files and the Docker environment.

## Monitoring and Accessing Job Status
1. Checking Job Deployment: Introduces the manuscript-cli list command to check the status of deployed jobs, providing an overview of job progress.
2. Viewing Job Logs: Explains the use of manuscript-cli logs command to access detailed logs for specific jobs, allowing for in-depth analysis of job execution.
3. GraphQL Interface: Guides users to the GraphQL address for querying the selected dataset, emphasizing the ability to select specific fields and execute queries.

## Advanced Data Parsing Techniques
1. Introduction to Contract Parsing: Introduces the concept of parsing blockchain contract data for more complex analyses, emphasizing the need for advanced techniques.
2. Accessing the GitHub Repository: Directs users to the Manuscript GitHub repository, emphasizing the availability of example scripts for guidance.
3. Editing and Deploying: Explains the process of customizing the manuscript.yaml file and using the manuscript-cli deploy command for local deployment.
4. Accessing Decoded Data: Highlights the ability to access decoded contract data through the same GraphQL interface, providing a comprehensive approach to data analysis.

## Job Management
1. Stopping Manuscript Jobs: Provides instructions on using the manuscript-cli stop command to halt running jobs, allowing users to manage active processes effectively.

**Comment by lxcong** [comment]
In addition to the GraphQL interface, can also add the interactive interface of AI Chat to access data?

**Reply by Lukisong82** [reply]
Did you mean the interface of using OpenAI, Gemini and other compatible AI models?

**Comment by KagemniKarimu** [comment]
This is a great idea, @Lukisong82 . I will be very eager to work on this as soon as I close a few more of the existing issues with the CLI. Until then, making a guide maybe counter-productive as there will likely be frequent breaking changes and it will go out of date frequently. :fire:
